src/screens/profile/editProfile.js

```diff
@@ -0,0 +1,234 @@
+import {
+  View,
+  ImageBackground,
+  StyleSheet,
+  ScrollView,
+  TouchableOpacity,
+  StatusBar,
+} from "react-native"
+import React, {useEffect, useState} from "react"
+import Header from "@components/header"
+import LeftComponent from "./components/editProfile/leftComponent"
+import fonts from "@assets/fonts"
+import {useNavigation, useTheme} from "@react-navigation/native"
+import Button from "@components/button"
+import {useForm} from "react-hook-form"
+import {launchImageLibrary} from "@utils/imagePicker"
+import {Divider, Icon} from "@rneui/themed"
+import useAuth from "@hooks/useAuth"
+import {findUserById, updateUser} from "@services/user"
+import {getImageByFileName, uploadImageByUri} from "@services/image"
+import SuccessDialog from "@components/successDialog"
+import LoadingDialog from "@components/loadingDialog"
+import {homeTabs} from "@common/navigator"
+import {Material} from "@common/icon"
+import NameContainer from "./components/editProfile/nameContainer"
+import PhoneContainer from "./components/editProfile/phoneContainer"
+import EmailContainer from "./components/editProfile/emailContainer"
+import AboutContainer from "./components/editProfile/aboutContainer"
+import WebsiteContainer from "./components/editProfile/websiteContainer"
+
+const EditProfile = () => {
+  const {colors} = useTheme()
+  const styles = makeStyles(colors)
+  const {userInfo} = useAuth()
+  const [profile, setProfile] = useState({})
+  const navigation = useNavigation()
+  const [image, setImage] = useState({
+    uri: "https://espclarity.com/wp-content/uploads/2021/07/3rd-member.png",
+    fileName: null,
+  })
+  const [dialog, setDialog] = useState({
+    isSuccess: false,
+    isLoading: false,
+  })
+
+  const {
+    handleSubmit,
+    control,
+    setValue,
+    formState: {errors},
+  } = useForm({
+    defaultValues: {
+      fullName: "",
+      phoneNumber: "",
+      email: "",
+      about: "",
+      website: "",
+    },
+  })
+
+  useEffect(() => {
+    fetchUser()
+    return () => {
+      setProfile({})
```

Comment:
DO NOT set state in clean up function, should remove all setState in clean up function